packages/codemods/src/transforms/v2/web-react/__testfixtures__/modal-isexpandedonmobile-prop.output.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+// @ts-ignore: No declaration -- The library is not installed; we don't need to install it for fixtures.
+import { ModalDialog } from '@lmc-eu/spirit-web-react';
+
+export const MyComponent = () => (
+  <>
+    <ModalDialog isExpandedOnMobile={false} />
+    <ModalDialog isExpandedOnMobile={true} />
```

Comment:
I think the output should be `<ModalDialog />` when it is set by default. Or just `<ModalDialog isExpandedOnMobile />`. It is possible?

Reply:
added